src/main/java/worker/Frontend.java

```diff
@@ -19,8 +20,12 @@
 
 
   ActorRef masterProxy = getContext().actorOf(
-      ClusterSingletonProxy.defaultProps("/user/master/active", "backend")
-      , "masterProxy");
+      ClusterSingletonProxy.props(
+          "/user/master",
+          ClusterSingletonProxySettings.create(getContext().system())
+              .withSingletonName("active")
```

Comment:
Actually, looking at this now, I can't find a `ClusterSingletonProxy.defaultProps` method in 2.4.0 at all, what am I missing?